.github/workflows/main.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     - run: rustup component add rustfmt
     - run: cargo fmt --all --check
     - run: |
-        for manifest in `find crates benches/benchsuite benches/capture -name Cargo.toml`
+        for manifest in `find benches/benchsuite benches/capture crates/resolver-tests -maxdepth 1 -name Cargo.toml`
         do
           echo check fmt for $manifest
           cargo fmt --all --manifest-path $manifest --check
```

Comment:
Why not just run `cargo fmt` in the root?

Reply:
Yep It should just work if we include those crate as workspace members. I'll include them in the next update btw.